.ci/install_kubernetes.sh

```diff
@@ -11,22 +11,69 @@ set -o pipefail
 
-if [ "$ID" != "ubuntu" ]; then
-        echo "Currently this script only works for Ubuntu. Skipped Kubernetes Setup"
+if [ "$ID" != "ubuntu" ] && [ "$ID" != "centos" ]; then
+        echo "Currently this script does not work on $ID. Skipped Kubernetes Setup"
         exit
 fi
 
-sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
-deb http://apt.kubernetes.io/ kubernetes-xenial-unstable main
-EOF"
-curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-chronic sudo -E apt update
+if [ "$KATA_HYPERVISOR" == "firecracker" ]; then
+	echo "Kubernetes will not work with $KATA_HYPERVISOR"
+	exit
+fi
+
+if [ "$ID" == "ubuntu" ]; then
+	sudo bash -c "cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+	deb http://apt.kubernetes.io/ kubernetes-xenial-unstable main
+	EOF"
+
+	chronic sudo -E sed -i 's/^[ \t]*//' /etc/apt/sources.list.d/kubernetes.list
+	chronic sudo -E sed -i '$ d' /etc/apt/sources.list.d/kubernetes.list
+	curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+	chronic sudo -E apt update
+	chronic sudo -E apt install --allow-downgrades -y kubelet="$kubernetes_version" kubeadm="$kubernetes_version" kubectl="$kubernetes_version"
+elif [ "$ID" == "centos" ]; then
+	sudo yum versionlock docker-ce
+
+	sudo bash -c "cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+	[kubernetes]
+	name=Kubernetes
+	baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
+	enabled=1
+	gpgcheck=1
+	repo_gpgcheck=1
+	gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+	EOF"
 
-chronic sudo -E apt install --allow-downgrades -y kubelet="$kubernetes_version" kubeadm="$kubernetes_version" kubectl="$kubernetes_version"
+	chronic sudo -E sed -i 's/^[ \t]*//' /etc/yum.repos.d/kubernetes.repo
+	chronic sudo -E sed -i '$ d' /etc/yum.repos.d/kubernetes.repo
+	install_kubernetes_version=$(echo $kubernetes_version | cut -d'-' -f1)
+	chronic sudo -E yum -y update
+	chronic sudo -E yum install -y kubelet-"$install_kubernetes_version" kubeadm-"$install_kubernetes_version" kubectl-"$install_kubernetes_version" --disableexcludes=kubernetes
+
+	# Disable selinux
+	if  [ "$(getenforce)" != "Disabled" ]; then
+		chronic sudo -E setenforce 0
+		chronic sudo -E sed -i 's/^SELINUX=enforcing/SELINUX=disabled/g' /etc/sysconfig/selinux
+	fi
+
+	# Packets traversing the bridge should be sent to iptables for processing
+	echo br_netfilter | sudo -E tee /etc/modules-load.d/k8s.conf
+	sudo -E modprobe -i br_netfilter
+	sudo -E bash -c 'echo "net.bridge.bridge-nf-call-ip6tables = 1" > /etc/sysctl.d/k8s.conf'
+	sudo -E bash -c 'echo "net.bridge.bridge-nf-call-iptables = 1" >> /etc/sysctl.d/k8s.conf'
+	sudo -E bash -c 'echo "net.ipv4.ip_forward = 1" >> /etc/sysctl.d/k8s.conf'
+	sudo -E sysctl --system
+
+	sudo -E systemctl enable kubelet
+else
+	echo "Kubernetes configuration not done on $ID"
+	exit
```

Comment:
This should be an error condition as the distro was checked at the top of the script. Hence, I think this should be a `die` call or `echo` to `stderr` and `exit 1`.